src/docs/fundamentals/ElectronicLabNotebooks.md

```diff
@@ -1,14 +1,359 @@
+  - mixed types of protocols
+  - some routine, some continuously evolving
+  - some protocols form the base to very central experiments and assays
+  - Protocols are an integral part of publications (*materials and methods* section). Methods are shared with peers
+5. classical pen-and-paper lab note book
+   1. metadata knowledge stuck in a single copy
+   2. hard to access (hidden in a shelf)
+   3. hard to maintain (over generations of researchers)
+   4. hard to communicate adaptations (stickers, post-its, hand-written notes) -->
+
+<!--
+
+In protocols we document exactly how a planned lab process was followed for a dedicated set of samples.
+While a well established method for RNA extraction may state that "between 20 and 30 mg" of a ground leaf shall be used as the starting material, we would denote that RNA was extracted from exactly "23.6 mg" of leaf material for one identifiable sample.
+While the method may generally recommend five technical replicates, the protocol would associate the five replicates with information
+
+
+Definitions:
+
+Method: how an experiment could be done (compare: recipe for baking a cake)
+Protocol: how an experiment was done (used this type of flour, baked for 45 instead of 40 minutes)
+
+
+the fact that the digitalized form of what used to be on **one** paper can now be replicated with ease facilitates quicker collaboration, sharing, and reuse of those central entities at least within
+
+
+
+
+
+Tools to edit and share protocols
+
+With the purpose of protocol editors
+
+1. the closed lab context
+2. the exchange on laboratory protocols
+3. ELNs d
+
+
+
+- good tool to digitalize your protocols
+- good protocol editor
+- allows free / flexible editing of protocols
+- sharing protocols with close surrounding / your lab
+  - easy to reuse
+- time stamping
+
+
+
+<!-- Where is the bridge between ELNs and ARCs? (ELNs = Protocols = Metadata) -->
+
+<!-- integration (Using an ELN with your ARC) -->
+<!-- you can take to make pave the way towards ELN-ARC integration (Using an ELN with your ARC) -->
+
+<!-- - the ARC is your place to structure, store, annotate, share and evolve all data to your research project -->
+
+<!-- 1:1 Project : ARC -->
+
+<!-- This guide was inspired by the [guide][web-nfdi4chem] made by the nfdi4chem consortia. -->
+
+<!-- References -->
+
+[knowledge-fair]: ../fundamentals/FairDataPrinciples.html "Article about FAIR Principles"
+[knowledge-mets]: ../fundamentals/Metadata.html "Article about Metadata"
+[web-elabftw]: https://www.elabftw.net/ "Homepage of eLabFTW"
+[web-elabdemo]: https://demo.elabftw.net/ "Demo of eLabFTW"
+[web-listerdemo]: https://elabftw.pharm.hhu.de/ "Demo of eLabFTW with LISTER"
+[web-elnfinder]: https://eln-finder.ulb.tu-darmstadt.de/home "ELN Finder"
+[web-elabjournal]: https://www.elabnext.com/products/elabjournal/ "Homepage of eLabJournal"
+[web-nfdi4chem]: https://knowledgebase.nfdi4chem.de/knowledge_base/docs/eln/ "ELN Guide made by nfdi4chem"
+
+---
```

Comment:
Ok, this is odd. The article starts new here with the YAML header. 
This might be due to moving the article at some point in the past months. 

Maybe remove my earlier notes from your copy and just work in your additions to the existing article. Sorry for the mess.

Reply:
i removed the stuff and created a new pull request. It should work now hopefully ^^